packages/cli/src/templates/nextjs-[pages].tsx

```diff
@@ -0,0 +1,62 @@
+import { useRouter } from 'next/router';
```

Comment:
Potentially, make this a `jsx` file so it works for JS only projects. No types are being declared anywhere here anyways

Reply:
yup, good point, plan is to add both!